main.c

```diff
@@ -6,7 +6,9 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <time.h>
 #include <unistd.h>
+#include <linux/limits.h>
```

Comment:
Oh, I missed that include. This is Linux-specific, the portable way to do it is:

```
#define _POSIX_C_SOURCE 1
#include <limits.h>
```

Reply:
Updated. The file already had `#define _POSIX_C_SOURCE 200809L`(POSIX.1-2008) so I left as is since it's newer than POSIX.1-1990.